### PR TITLE
Update eksctl nodegroup command

### PR DESCRIPTION
**Pull request by realgaurav**
## Description

Add `max-pods-per-node` option to `eksctl` command on EKS installation page.

## Release Note

```release-note
None required
```

### CODE REVIEW

**Comment by caseydavenport** [comment]
Deploy preview for *calico* ready!

Built with commit 627b9729dbc9a2c168164af61053a65431e6d14b

https://deploy-preview-3629--calico.netlify.app

File: getting-started/kubernetes/managed-public-cloud/eks.md [hunk 1]
```diff
@@ -53,10 +53,10 @@ Before you get started, make sure you have downloaded and configured the {% incl
 1. Finally, add nodes to the cluster.
 
    ```bash
-   eksctl create nodegroup --cluster my-calico-cluster --node-type t3.medium --node-ami auto
+   eksctl create nodegroup --cluster my-calico-cluster --node-type t3.medium --node-ami auto --max-pods-per-node 100
    ```
 
-   > **Tip**: See `eksctl create nodegroup --help` for the full set of node group options.
+   > **Tip**: In absence of `--max-pods-per-node` option above, EKS will limit {% include open-new-window.html text='number of pods based on node-type' url='https://github.com/awslabs/amazon-eks-ami/blob/master/files/eni-max-pods.txt' %}. See `eksctl create nodegroup --help` for the full set of node group options.
```
Comment (caseydavenport) [inline getting-started/kubernetes/managed-public-cloud/eks.md:59]: "In _the_ absence of" (or just "without")

also "_the_ number of pods"

File: getting-started/kubernetes/managed-public-cloud/eks.md [hunk 2]
```diff
@@ -53,10 +53,10 @@ Before you get started, make sure you have downloaded and configured the {% incl
 1. Finally, add nodes to the cluster.
 
    ```bash
-   eksctl create nodegroup --cluster my-calico-cluster --node-type t3.medium --node-ami auto
+   eksctl create nodegroup --cluster my-calico-cluster --node-type t3.medium --node-ami auto --max-pods-per-node 100
    ```
 
-   > **Tip**: See `eksctl create nodegroup --help` for the full set of node group options.
+   > **Tip**: Without `--max-pods-per-node` option above, EKS will limit the {% include open-new-window.html text='number of pods based on node-type' url='https://github.com/awslabs/amazon-eks-ami/blob/master/files/eni-max-pods.txt' %}. See `eksctl create nodegroup --help` for the full set of node group options.
```
Comment (caseydavenport) [inline getting-started/kubernetes/managed-public-cloud/eks.md:59]: Should be "without the"

Comment (realgaurav) [reply getting-started/kubernetes/managed-public-cloud/eks.md:59]: changed to _Without the_

**Pull request merged**